DATA_SOURCES.md

```diff
@@ -191,6 +191,9 @@ For many European countries, data is available from [ENTSO-E](https://transparen
   [wikipedia.org](https://en.wikipedia.org/wiki/List_of_generating_stations_in_Canada)
 - Canada (Alberta): [AESO](http://ets.aeso.ca/ets_web/ip/Market/Reports/CSDReportServlet),
   [wikipedia.org](https://en.wikipedia.org/wiki/List_of_generating_stations_in_Alberta)
+- Canada (Manitoba)
```

Comment:
Just a really small note to remove Manitoba from 4 lines up from here where it says its capacities come from Wikipedia along with a bunch of other provinces